routing/message/handlers/passthrough/command_handler.go

```diff
@@ -25,34 +25,30 @@ const commandHandlerName = "passthrough_command_handler"
 
 // A simple command passthrough handler that forwards all cloud-to-device messages from Azure IoT Hub to local MQTT broker on a preconfigured topic.
 type commandHandler struct {
-	topics string
+	topic string
 }
 
 // CreateCommandHandler instantiates a new command handler that forward cloud-to-device messages to the local message broker using the given topic.
-func CreateCommandHandler() handlers.MessageHandler {
-	return &commandHandler{}
+func CreateCommandHandler(topic string) handlers.CommandHandler {
+	return &commandHandler{
+		topic: topic,
+	}
 }
 
-// Init gets the local topic to publish the messages.
-func (h *commandHandler) Init(settings *config.AzureSettings, connSettings *config.AzureConnectionSettings) error {
-	h.topics = settings.PassthroughCommandTopic
+// Init does nothing.
+func (h *commandHandler) Init(connInfo *config.CloudConnectionInfo) error {
 	return nil
 }
 
 // HandleMessage creates a new message with the same payload as the incoming message and sets the configured local topic to publish it.
 func (h *commandHandler) HandleMessage(msg *message.Message) ([]*message.Message, error) {
 	msgID := watermill.NewUUID()
 	outgoingMessage := message.NewMessage(msgID, msg.Payload)
-	outgoingMessage.SetContext(connector.SetTopicToCtx(outgoingMessage.Context(), h.topics))
+	outgoingMessage.SetContext(connector.SetTopicToCtx(outgoingMessage.Context(), h.topic))
```

Comment:
Do we need to check that the topic is not comma-separated.

Reply:
If any checks are to be implemented, these should be done in the initialization phase when settings are loaded from flags or config file.